docs/test-results/viewing-test-results.md

```diff
@@ -44,7 +45,7 @@ To view automated test results:
 
 2. On the **Test Results** page, click the **Virtual Cloud** or **Real Devices** radio button to view the relevant device results.
 
```

Comment:
```suggestion
```

Procedure steps should have single-line spacing (current output shown below). 

<img width="828" alt="Screen Shot 2022-10-06 at 3 22 23 PM" src="https://user-images.githubusercontent.com/15352964/194401130-f8ae3fde-eb08-4212-a645-1638dd80dd6a.png">